### PR TITLE
Fix to add opening area for windows to IVRS report.

### PR DESCRIPTION
**Pull request by JasonGlazer**
Addresses #4605 to add the window opening to the Input Verification and Summary report Zone Summary table. The opening area includes the glass area of the window plus the frame.
- [X] Code style (parentheses padding, variable names)
- [X] Functional code review (it has to work!)
- [X] If defect, results of running current develop vs this branch should exhibit the fix
- [x] CI status: all green or justified
- [x] Performance: CI Linux results include performance check -- verify this
- [x] Unit Test(s)
- [x] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by JasonGlazer** [comment]
@Myoldmopar I noticed that not all the CI tests were run for some reason. It is not that important to rerun for the previous commit since I still need to add a unit test anyway.

**Comment by Myoldmopar** [comment]
Yeah i should go back over all our machines and get them all updated and
rebooted anyway.

**Comment by JasonGlazer** [comment]
@Myoldmopar  and @mjwitte this is passing all unit tests and is ready to be reviewed and merged.

**Comment by nrel-bot-2** [comment]
@JasonGlazer @lgentile it has been 14 days since this pull request was last updated.

File: src/EnergyPlus/OutputReportTabular.cc [hunk 1]
```diff
@@ -9708,6 +9714,7 @@ namespace OutputReportTabular {
 								windowAreaW += curArea * mult;
 								if ( isConditioned ) windowAreaWcond += curArea * mult;
 							}
+							zoneOpeningArea( zonePt ) += curArea * mult; // total window opening area for each zone (glass plus frame area)
```
Comment (mjwitte) [inline src/EnergyPlus/OutputReportTabular.cc]: @JasonGlazer multiplier is applied here and below - looks like it gets multiplied twice?  Will make some test files to confirm.

**Comment by mjwitte** [comment]
@JasonGlazer Added four new test files to DefectFiles\4605.  The opening area is incorrectly multiplied once in the zone-by-zone rows (all the other columns are unmultiplied) and then it is multiplied by 100 instead of 10 in the total rows.

**Comment by mjwitte** [comment]
The zone multiplier issue is fixed.  But now I see a discrepancy in the `9294-building1 with underground parking_V850` defect file which has no zone multipliers but it has _window_ multipliers.  The total opening area in the window-wall ratio subtable includes the window multipliers (total opening area is 707.9).  But the Zone Summary table is not accounting for the window multipliers (total opening area is 324.5).  The window multipliers should be applied both in the zone-by-zone rows and in the total.

**Comment by Myoldmopar** [comment]
This is still on hold then, right @JasonGlazer?  Cycling through my Pivotal tickets and I made a build of this branch to verify, then noticed @mjwitte's comment.

**Comment by JasonGlazer** [comment]
@Myoldmopar Yes, let me take a look at the issue that Mike found.

**Comment by JasonGlazer** [comment]
@mjwitte I added some more test files to look at cases with both zone and window multipliers.

**Comment by JasonGlazer** [comment]
@mjwitte The CI tests are all clear. Please take another look at it.

**Comment by mjwitte** [comment]
Re-tested and all looks good.

**Pull request merged**